src/igvfd/types/file_set.py

```diff
@@ -504,11 +508,16 @@ def summary(self, request, file_set_type, input_file_sets=[], files=[], samples=
             if not (assay_terms and all(x in crispr_screen_terms for x in assay_terms)):
                 file_set_type_phrase = ', '.join(fileset_types)
 
+        control_phrase = ''
+        if len(control_type_set) > 0:
+            control_phrase = f'with {", ".join(control_type_set)} control'
```

Comment:
Should this be pluralized for multiple control_type? Can you also add a test for multiple control_type?